proposals/stage-3/CPC_CHARTER/CPC-CHARTER.md

```diff
@@ -0,0 +1,320 @@
+# Cross Project Council (CPC) Charter
+
+## Section 1. Guiding Principle.
+
+The OpenJS Foundation will operate transparently, openly,
+collaboratively, and ethically. Project proposals, timelines, and status
+must not merely be open, but also easily visible to outsiders.
+
+## Section 2. Evolution of OpenJS Foundation Governance.
+
+Most large, complex open source communities have both a business and a
+technical governance model. OpenJS Foundation’s technical leadership
```

Comment:
This first sentence of this paragraph should be omitted IMO.

Reply:
I think we should change that in a follow up PR